### DISCUSSION TITLE
Is caub patched

### DISCUSSION THREAD

**Discussion by rolexwatch123** [3kh0/ext-remover#961] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
I used caub on 119 but low after a few months it somehow updated to 121 and I cannot access skiovox anymore

**Comment by AshtonDavies** [comment, marked as answer]
It very well could be patched, as you're not supposed to set metered by the importing of an .onc file.

Many people are reporting here lately that it quit working for them, so I think it is patched.

Luckily we can freeze updates with a DNS.

EDIT: Well, the other possibility is that administrators are pushing updates manually, which is a check that is probably not skipped even on metered networks.

**Reply by rolexwatch123** [reply]
how do i do a DNS

**Reply by xoidex** [reply]
just curious but how does caudns work?

**Reply by rolexwatch123** [reply]
how do i do a dns

**Reply by AshtonDavies** [reply]
> how do i do a DNS

[cauDNS](https://caudns.vercel.app)

**Reply by rolexwatch123** [reply]
its blocked

**Comment by AshtonDavies** [comment]
Did you happen to connect to an un-CAUBed network?

It's a good idea to check that it is working. If you import the ONC file and it doesn't set the network to metered, then you can probably confirm it's patched.

**Reply by rolexwatch123** [reply]
no i didnt connect i was still on the school wifi

**Reply by AshtonDavies** [reply]
> no i didnt connect i was still on the school wifi

Please check in chrome://version if it shows the red message.

**Reply by rolexwatch123** [reply]
it still does show the red warning in chromeos settings but not in chrome://version

**Reply by AshtonDavies** [reply]
> it still does show the red warning in chromeos settings but not in chrome://version

Okay, then that narrows down to one possibility, which is that administrators are force-updating Chromebooks.

They do this when they notice Chromebooks being actively used on old versions, as it may stick out in their devices panel in the Admin Console.

I think you can use cauDNS or OmadaDNS to prevent updates and maybe administrator connectivity. You can check the [OmadaDNS repository](https://codeberg.org/omada/config/src/branch/main/dns) to see what's in the blocklist.

**Reply by rolexwatch123** [reply]
network state is blocked for me

**Reply by AshtonDavies** [reply]
> network state is blocked for me

You do not need chrome://network#state to set a DNS.

**Reply by rolexwatch123** [reply]
what do i do

**Reply by AshtonDavies** [reply]
> what do i do

In chrome://os-settings, in your network settings, there just be a more settings drop-down, and a DNS configuration that you can switch to custom name servers.

**Reply by rolexwatch123** [reply]
what name servers do i use

**Reply by AshtonDavies** [reply]
> what name servers do i use

I have not used this yet to know if/how it works, but set it to **66.94.105.229**.

Then refresh your network and you should be connected to OmadaDNS.

**Reply by rolexwatch123** [reply]
set all of them to this?

**Reply by AshtonDavies** [reply]
> set all of them to this?

Just one needs to be set. The other three are extra servers, so it doesn't really make a difference.

**Reply by rolexwatch123** [reply]
what does omadadns do

**Reply by AshtonDavies** [reply]
> what does omadadns do

Oh, I guess it blocks extensions and apps pretty much.

Well, if you can't use cauDNS because of chrome://network#state being blocked, then use mine I guess. 🤷

94.140.14.49

https://linkip.adguard-dns.com/linkip/d4cfbd43/MrXXtcXx4U4QvPOyloGK4Y3gdER6FjJrYAiHdQoehux

**Reply by rolexwatch123** [reply]
ok

**Reply by rolexwatch123** [reply]
where do i put the link

**Reply by AshtonDavies** [reply]
It should block policy updates and software updates.

**Reply by rolexwatch123** [reply]
do i click the link

**Reply by AshtonDavies** [reply]
> do i click the link

Yes

**Reply by rolexwatch123** [reply]
none of the extensions turned off except smoothwall
